CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# Change Log
+
+All notables changes to this project are documented in this file.
+
+The format is inspired by [Keep a Changelog].
+
+[Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
+
+<!-- NOTE: towncrier will not alter content above the TOWNCRIER line below. -->
+
+<!-- TOWNCRIER -->
+
+## 19.1 (2019-12-20)
+
+### Deprecations and Removals
+
+- Remove staking-related roothash messages.
+  ([#2377](https://github.com/oasislabs/oasis-core/issues/2377))
+
+  There is no longer a plan to support direct manipulation of the staking accounts
+  from the runtimes in order to isolate the runtimes from corrupting the
+  consensus layer.
+
+  To reduce complexity, the staking-related roothash messages were removed. The
+  general roothash message mechanism stayed as-is since it may be useful in the
+  future, but any commits with non-empty messages are rejected for now.
+
+- Remove `oasis-node debug dummy` sub-commands.
+  ([#2492](https://github.com/oasislabs/oasis-core/issues/2492))
+
+  These are only useful for testing, and our test harness has a internal Go API
+  that removes the need to have this functionality exposed as a sub-command.
+
+### Features
+
+- Add `oasis-node debug storage export` sub-command.
+  ([#1845](https://github.com/oasislabs/oasis-core/issues/1845))
+
+- Add `oasis-node unsafe-reset` sub-command which resets the node back to a
+  freshly provisioned state, preserving any key material if it exists.
+  ([#2435](https://github.com/oasislabs/oasis-core/issues/2435))
+
+- Add consensus block and transaction metadata accessors.
+  ([#2482](https://github.com/oasislabs/oasis-core/issues/2482))
+
+  In order to enable people to build "network explorers", we exposed some
+  additional methods via the consensus API endpoint, specifically:
+
+  - Consensus block metadata.
+  - Access to raw consensus transactions within a block.
+  - Stream of consensus blocks as they are finalized.
+
+### Bug Fixes
+
+- Reduce Badger in-memory cache sizes.
+  ([#2484](https://github.com/oasislabs/oasis-core/issues/2484))
+
+  The default is 1 GiB per badger instance and we use a few instances so this
+  resulted in some nice memory usage.
+
+
+## 19.0 (2019-12-18)
+
+- Initial release.
```

Comment:
can we have this not say 19.0 (2019-12-18) is the initial release? it's not the initial release. if we don't want to go back and retroactively describe what changed between a9a532e6be67512b3e4e581b5a57a97dff5715ca and 19.0, it's less confusing not to have this section at all

Reply:
Agreed, this is not ideal. I would still like to have this section so that readers will know what the Change Log section for the next release, i.e. 19.1, refers to.

I've changed the contents to actually describe a process change, i.e. that the project started using a new versioning and release process and adopted a [CalVer](http://calver.org) (calendar versioning) scheme. The latter also explains why this release has version 19.0 as opposed to 1.0 or similar.

Please, take another look.

Reply:
thx for the changes